### PR TITLE
repo: Fix shellcheck errors (in newer version)

### PR DESCRIPTION
**Pull request by phlax**
Signed-off-by: Ryan Northey <ryan@synca.io>

Partial fix for #20199 

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Comment by phlax** [comment]
cc @kfaseela

File: test/extensions/transport_sockets/tls/ocsp/test_data/certs.sh [hunk 1]
```diff
@@ -5,7 +5,8 @@
 set -e
 
 readonly DEFAULT_VALIDITY_DAYS=${DEFAULT_VALIDITY_DAYS:-730}
-readonly HERE=$(cd "$(dirname "$0")" && pwd)
+readonly HERE
+HERE=$(cd "$(dirname "$0")" && pwd)
```
Comment (rojkov) [inline test/extensions/transport_sockets/tls/ocsp/test_data/certs.sh:9]: shouldn't it be the other way around? assign first, then mark it readonly?

Comment (phlax) [reply test/extensions/transport_sockets/tls/ocsp/test_data/certs.sh:9]: i wasnt sure about that - but shellcheck didnt complain at this - i can try the other way...

Comment (phlax) [reply test/extensions/transport_sockets/tls/ocsp/test_data/certs.sh:9]: works setting first - my intuition was that readonly in bash is set once - but ill swap these around

**Review by kfaseela — APPROVED** [review]

**Review by rojkov — APPROVED** [review]
Thanks! LGTM

**Pull request merged**